l10n_nl_xaf_auditfile_export/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "XAF auditfile export",
+    "version": "14.0.1.0.0",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/l10n-netherlands",
+    "license": "AGPL-3",
+    "category": "Invoices & Payments",
```

Comment:
Please Change category so it shows up corretly in odoo appstore and backend applist.
```"category": "Localization",```
or
```"category": "Localization\Netherlands",```